docs/examples/ensemble-window.ipynb

```diff
@@ -0,0 +1,429 @@
+    "rand_indexes = {}  # to keep track of the performances\n",
+    "\n",
+    "for cost in cost_functions:\n",
+    "    start_time = time.time()\n",
+    "\n",
+    "    algo = rpt.Window(width=window_size, model=cost, jump=1).fit(signal)\n",
+    "    predicted_bkps = algo.predict(n_bkps=n_bkps)\n",
+    "\n",
+    "    print(f\"{cost} cost computation in {time.time() - start_time:.3f} s\")\n",
+    "\n",
+    "    rand_indexes[cost] = np.around(randindex(bkps, predicted_bkps), 3)\n",
+    "    scores.append(algo.score)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have the performance of the classical method on each cost function, let's dive into the ensemble method. We will then compare the results."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "qB3Nhz3njORl"
+   },
+   "source": [
+    "### Ensemble method: multiple costs for one prediction\n",
+    "\n",
+    "The paper we rely on [Katser2021](#Katser2021), uses the score from each cost function to detect the changepoints. This is why we stored them in the previous section."
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "!!! note\n",
+    "    As a reminder, [here](../user-guide/detection/window.md) is how the score is computed for the window-based search method, it can also be called the *discrepancy measure*."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's see how those scores look like. In the following cell, each row represents the score for a specific cost function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 729
+    },
+    "id": "nBk9sihcLk2d",
+    "outputId": "90bf9753-3ca1-43fd-a3c9-d103c5a923a0"
+   },
+   "outputs": [],
+   "source": [
+    "scores = np.array(scores).T\n",
+    "\n",
+    "# Display the scores with the change points to predict\n",
+    "append_scores = np.append(\n",
+    "    np.ones((window_size // 2, len(cost_functions))) * float(\"inf\"), scores, axis=0\n",
+    ")\n",
+    "_ = rpt.display(append_scores, bkps)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The idea of the CPD Ensemble method is to scale and aggregate the scores with some well chosen scaling and aggregation function to get new score which will allow us to get a better prediction of the changepoints.\n",
+    "\n",
+    "In the paper, they perform multiple tests on several scaling and aggregation functions on two datasets. In turned out that the *MinAbs* scaling function and the *WeightedSum* aggregation function worked the best.\n",
+    "\n",
+    "The *MinAbs* function is defined as follows:\n",
+    "For $s$ a timeseries, \n",
+    "\n",
+    "$$\n",
+    "\\textit{MinAbs}(s)_i = \\frac{s_i}{|\\min_{j}{s_j}|}\n",
```

Comment:
```suggestion
    "\\textit{MinAbs}(s_i) = \\frac{s_i}{|\\min_{j}{s_j}|}\n",
```

Reply:
I think it is a question of notation but for me writing: 
```Latex
$\textit{MinAbs}(s_i) = \frac{s_i}{|\min_{j}{s_j}|}$
```
is an abuse of notation since `\textit{MinAbs}` would only have the value of `s_i` as input. Thus it would not be able to compute `\min_{j}{s_j}`.

If it is usually defined the way you corrected it, then I don't mind changing.

Reply:
That is fine by me the way you write it. Maybe when you introduce `For $s$ a timeseries,`, try to also tell that it is subscripted by `i`. That might help the reader in my opinion. 
Let's resolve this conversation and we will see if Charles has an opinion on this notation !